open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/view-generator-connectors/pom.xml

```diff
@@ -24,5 +24,30 @@
 
     <artifactId>view-generator-connectors</artifactId>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>open-connector-framework</artifactId>
+            <version>${open-metadata.version}</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>repository-services-apis</artifactId>
+            <version>${open-metadata.version}</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.derby</groupId>
+            <artifactId>derbyclient</artifactId>
+            <version>10.14.1.0</version>
```

Comment:
The deprecated virtualizer used a variable derby.version
The gaian connector is using 10.11.1.1
10.14 is much newer, but I have previously noticed some compatability issues. I would be inclined to be consistent and go with the version shipped by gaian - though if tested and happy that's ok.
If so, worth modifying the other versions & moving this hard version number to an appropriate property